homework/list.js

```diff
@@ -1,44 +1,51 @@
 'use strict';
 
-(function() {
+(function(exports) {
 
-  var _listNoteContent = [];
-  var _wrapper = document.querySelector('#note-list-wrapper');
+var ListManager = function() {
+  this._listNoteContent = [];
+  this._wrapper =document.querySelector('#note-list-wrapper');
+};
 
-  function start() {
-    fetchList(function(data) {
-      updateList(data);
-      drawList();
-      preloadFirstNote();
-    });
-    window.addEventListener('click', function(event) {
-      onNoteOpen(event);
-    });
-  }
+ListManager.prototype = {
+
+  start() {
+      this.fetchList().then((function(response) {
```

Comment:
nit: I recommend to use Arrow function rather than just function binding, although they're both legit.
See:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#Lexical_this